main.py

```diff
@@ -2,6 +2,55 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
+
+genres = ["Western", "Action", "Dramma"]
+actors = [
+    ("George", "Klooney"),
+    ("Kianu", "Reaves"),
+    ("Scarlett", "Keegan"),
+    ("Will", "Smith"),
+    ("Jaden", "Smith"),
+    ("Scarlett", "Johansson"),
+]
+
+
+def execute_create_query() -> None:
+    for genre in genres:
+        Genre.objects.create(
+            name=genre
+        )
+
+    for actor_name, actor_last_name in actors:
+        Actor.objects.create(
+            first_name=actor_name,
+            last_name=actor_last_name,
+        )
+
+
+def execute_update_query() -> None:
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(first_name="George",
+                         last_name="Klooney").update(
+        last_name="Clooney"
+    )
+
+    Actor.objects.filter(first_name="Kianu",
+                         last_name="Reaves").update(
+        first_name="Keanu",
+        last_name="Reeves"
+    )
```

Comment:
it's better to use the same style for breaking lines

Reply:
I just don't like look of separated `.update()` so didn't touch it, but it's still more readable after same breaking like styling, so I refactored that )